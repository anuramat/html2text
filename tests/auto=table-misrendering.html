<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 transitional//EN"><html><head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<title>%s</title>
</head>
<body>
<table cellpadding="4" style="border: 1px solid #000000; border-collapse: collapse;" border="1">
 <tr>
  <th bgcolor="#abebc6">New Diffs</th>
  <th bgcolor="#abebc6">Int Diffs</th>
  <th bgcolor="#abebc6">Size Diffs</th>
  <th bgcolor="#abebc6">Diffs</th>
  <th bgcolor="#abebc6">New Sucs</th>
  <th bgcolor="#abebc6">Sucs</th>
 </tr>
 <tr>
  <td align="right">0</td>
  <td align="right">0</td>
  <td align="right">0</td>
  <td align="right">0</td>
  <td align="right">0</td>
  <td align="right">56</td>
 </tr>
</table>
<h4>Lrg List</h4>
<table cellpadding="4" style="border: 1px solid #000000; border-collapse: collapse;" border="1">
 <tr>
  <th bgcolor="#abebc6">Lrg</th>
  <th bgcolor="#abebc6">New Diffs</th>
  <th bgcolor="#abebc6">Int Diffs</th>
  <th bgcolor="#abebc6">Size Diffs</th>
  <th bgcolor="#abebc6">Diffs</th>
  <th bgcolor="#abebc6">New Sucs</th>
  <th bgcolor="#abebc6">Sucs</th>
  <th bgcolor="#abebc6">Status</th>
 </tr>
 <tr>
  <td>lrgdbconcmrbdtool_bdcell</td>
  <td align="right">0</td>
  <td align="right">0</td>
  <td align="right">0</td>
  <td align="right">0</td>
  <td align="right">0</td>
  <td align="right">56</td>
  <td>completed</td>
 </tr>
</table>
