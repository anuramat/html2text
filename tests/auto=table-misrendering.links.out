auto=table-misrendering.default.out